creator-node/src/dbManager.js

```diff
@@ -37,6 +46,98 @@ class DBManager {
 
     return file.dataValues
   }
+
+  /**
+   * TODO document
```

Comment:
worth documenting that this is currently unused, but will be key later on for state reconciliation on secondaries thjat end up with non-contiguous block numbers

Reply:
good call, done

Reply:
is it? there appear to be TODOS here

Reply:
yeah they are but just as reminder for future work. not related to this PR. resolving as i have a note to cleanup and test this separately